lib/rucio/core/rule.py

```diff
@@ -2414,18 +2415,18 @@ def examine_rule(rule_id: str, *, session: "Session") -> dict[str, Any]:
 
 
 @transactional_session
-def get_evaluation_backlog(expiration_time: int = 600, *, session: "Session") -> tuple[int, datetime]:
+def get_evaluation_backlog(expiration_time: int = 600, *, session: "Session") -> Row[tuple[int, datetime]]:
```

Comment:
No, `Row[tuple[int, datetime]]` is not correct. `tuple[int, datetime]` is, but you may need to use `result._tuple()` to convince the type checker, see https://docs.sqlalchemy.org/en/20/core/connections.html#sqlalchemy.engine.Row._tuple.

Reply:
ignore the latest force-push, hadn't seen this comment - checking